.neutrinorc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         .rule('js-yaml')
           .test(/\.(yaml|yml)$/)
           .use('js-yaml-loader')
+          .options({safe: true})
```

Comment:
`.neutrinorc.js` is not linted since it's not part of the `src` directory. Could we add some spacing:

```js
.options({ safe: true })
```

Reply:
sure!